consulrunner/clusterrunner.go

```diff
@@ -63,10 +63,10 @@ func (cr *ClusterRunner) ConsulVersion() string {
 	session, err := gexec.Start(cmd, GinkgoWriter, GinkgoWriter)
 	Expect(err).NotTo(HaveOccurred())
 	Eventually(session).Should(gexec.Exit(0))
-	Expect(session.Out).To(gbytes.Say("Consul v"))
+	Expect(session.Out).To(gbytes.Say("Consul "))
 	lines := strings.Split(string(session.Out.Contents()), "\n")
 	versionLine := lines[0]
-	return strings.TrimPrefix(versionLine, "Consul v")
+	return strings.TrimPrefix(versionLine, "Consul ")
```

Comment:
This will break the cases where there is a 'v' and we intended to remove it (which would break `HasPerformanceFlag()` below, for example).

Could we instead do a nested trim? Something like this:

```strings.TrimPrefix(strings.TrimPrefix(versionLine, "Consul "), "v"))```

If you go that route, also please add a small comment explaining why we have this nested TrimPrefix since it looks kinda funny.